src/Cosi-BLS/mvp-election-beacon.lisp

```diff
@@ -331,7 +331,7 @@ based on their relative stake"
 
 (defun make-signed-call-for-election-message (pkey epoch skey)
   (let ((skel  (make-call-for-election-message-skeleton pkey epoch)))
-    (um:conc1 skel (pbc:sign-hash (hash/256 skel) skey))))
+    (um:append1 skel (pbc:sign-hash (hash/256 skel) skey))))
```

Comment:
What is this change, which is vaguely explained as ```Fix election requests (dbm)``` in the commit log message, doing here a commit for the config system.  It's by a different developer for a different system. Therefore, it does not belong in this PR.  The commit should be redone without if the were done together. I have not checked the history.

Reply:
> What is this change

This branch was intended to show that `emotiq/config` can be used to run a chain.  As such it contains code to add the genesis block.  

The change to fix election requests <https://github.com/emotiq/emotiq/pull/289/commits/00c71d3956c5cfe88994de768caa8c082bdf88bf> was noted by @dbmcclain as necessary for multiple elections to succeed.

Reply:
We just want small, coherent commits. This was a replacement for a previous PR, and it should address the issues that were there, which it largely has, not add new features or fix other bugs opportunistically.  If David's fix is needed, please do it as a separate PR, or have David do it.  The genesis one would be fine to do as a followup separate PR, but as I've told you I have slightly different commit/PR waiting for that, so I might as well do it.

Reply:
Oh I just noticed it looks like the above is actually a currently approved PR: Fixup: UM:CONC1 should have been UM:APPEND1 for non-destructive behavior #289 . Accordingtly, it should be put on dev, and this branch should be rebased on top of it.